packages/client/manifest.json

```diff
@@ -6723,7 +6723,21 @@
     "illegalChildren": ["section", "sidepanel"],
     "showEmptyState": false,
     "draggable": false,
-    "info": "Side panels are hidden by default. They will only be revealed when triggered by the 'Open Side Panel' action."
+    "info": "Side panels are hidden by default. They will only be revealed when triggered by the 'Open Side Panel' action.",
+    "sendEvents": true,
+    "settings": [
+      {
+        "type": "boolean",
+        "key": "clickOutsideToClose",
+        "label": "Click outside to close",
+        "defaultValue": true
```

Comment:
We can't default this to true, because all existing apps won't have this value set. For backwards compatibility we need to assume that the default value of any new settings are falsey, and invert the logic.

In this case, this new setting would have to something like "Ignore clicks outside" with a default value of false. This means all old apps will continue to work as normal and the new behaviour is opt in.